main.js

```diff
@@ -0,0 +1,430 @@
+$(document).ready(initializeGame);
+
+function initializeGame(){
+    attachClickHandlers();
+    currentPlayer = player[0];
+    $(".player1").find(".token1").hide();
+};
+function attachClickHandlers(){
+    $(".forfeitButton").on("click", resetGame);
+    $(".column").on("click", piecePlacement);
+
+};
+
+
+// GLOBAL VARIABLES
+
+var column_clicked = null;
+var row_clicked = null;
+var rowToPlacePiece = null;
+var putPieceInRow = 0;
+var games_played = 0;
+var attempts = 0;
+var piecesInColumn = 0;
+// ----------------
+
+
+var player = [];
+player[0] = {
+    name: "player1",
+    points: 0,
+    gamesWon: 0,
+    gamesLost: 0,
+    gamesTied: 0,
+};
+player[1] = {
+    name: "player2",
+    points: 0,
+    gamesWon: 0,
+    gamesLost: 0,
+    gamesTied: 0,
+};
+
+
+function placePlayerToken(piecesInRow) {
+    if (currentPlayer === player[0]) {
+        gameArray[piecesInRow][column_clicked].token = 1;
+    } else if (currentPlayer === player[1]) {
+        gameArray[piecesInRow][column_clicked].token = 2;
+    }
+}
+
+
+
+
+function displayToken(piecesInRow, column_clicked) {
+    if (gameArray[piecesInRow][column_clicked].token === 1) {
+        $(".row" + piecesInRow + " .column" + column_clicked).addClass("showPlayer1")
+
+    } else if (gameArray[piecesInRow][column_clicked].token === 2) {
+        $(".row" + piecesInRow + " .column" + column_clicked).addClass("showPlayer2")
```

Comment:
you could also select elements with custom attributes.

like if your element was 
``` <div row="4" column="3">```
you could select it with
```
$("div[column="+column_clicked+"][row="+piecesInRow+"]").addClass("showPlayer"+ currentPlayer.tokenID);